streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -161,6 +165,9 @@ public void before() throws Exception {
         props.put(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getPath());
         props.put(StreamsConfig.NUM_STREAM_THREADS_CONFIG, NUM_THREADS);
 
+        builder = new StreamsBuilder();
+        builder.stream("source");
```

Comment:
Can you create a method to `getBuilderWithSource()` and just call it inline instead of creating a new mutable field for all the tests to rely on?

Reply:
Good call, done